tasks/src/document-question-answering/data.ts

```diff
@@ -0,0 +1,57 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from the 2020 DocVQA challenge. The documents are taken from the UCSF Industry Documents Library.",
+			id:          "eliolio/docvqa",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Question",
+				content: "Which is the most critical issue for live rep support?",
+				type:    "text",
+			},
+			{
+				filename: "document-question-answering-input.jpeg",
+				type:    "img",
+			},
+		],
+		outputs: [
+			{
+				label:   "Answer",
+				content: "Product Quality/Liability Issues",
+				type:    "text",
+			},
+		],
+	},
+	metrics: [
+		{
+			description: "The evaluation metric for the DocVQA challenge is the Average Normalized Levenshtein Similarity (ANLS). This metric is robust to OCR errors and is based on the Levenshtein distance between the predicted answer and the ground truth answer.",
```

Comment:
```suggestion
			description: "The evaluation metric for the DocVQA challenge is the Average Normalized Levenshtein Similarity (ANLS). This metric is robust to OCR errors and is based on the Levenshtein distance between the predicted answer and the ground truth answer.",
```
In this pages we try to avoid using terms or try to simplify them if we will use them 🙂 to have a broader audience. Maybe avoid using abbreviations as well and write on long version?

Reply:
I chose to simplify it 😊